ns/install.sh

```diff
@@ -0,0 +1,2 @@
+sudo apt-get install ruby
+sudo gem install erubis
```

Comment:
Let's not worry about it now, as it's working for you and me, but I'll take care of this sometime after merge.